apps/fluent-tester/src/FluentTester/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -106,7 +106,12 @@ const tokenCheckbox: React.FunctionComponent = () => {
   return (
     <View>
       <HoverCheckbox label="A checkbox with checkmark visible on hover" onChange={onChangeUncontrolled} defaultChecked={false} />
-      <CircleColorCheckbox label="A circular token-customized checkbox" circular onChange={onChangeUncontrolled} defaultChecked={true} />
+      <CircleColorCheckbox
+        label="A circular token-customized checkbox"
+        shape="circular"
```

Comment:
late super-nit question, is using `circular` vs `circle` a particular decision for alignment or descriptive purposes?

Reply:
For me it's an alignment decision. Idk what the decision was on the FluentUI React side. It does align with the same property on the Button (which has a shape prop with the values rounded/circular/square)